Examples/Tests/reduced_diags/analysis_reduced_diags.py

```diff
@@ -76,90 +91,86 @@
-sum_weight_electron_data = NPdata[1][7]
-sum_weight_proton_data = NPdata[1][8]
-sum_weight_photon_data = NPdata[1][9]
-
-# PART3: print and assert
-
-max_diffEmax = max(abs(max_Exdata-max_Ex),abs(max_Eydata-max_Ey),
-                   abs(max_Ezdata-max_Ez),abs(max_Edata-max_E))
-max_diffBmax = max(abs(max_Bxdata-max_Bx),abs(max_Bydata-max_By),
-                   abs(max_Bzdata-max_Bz),abs(max_Bdata-max_B))
-max_diffrhomax = max(abs(max_rho_data-max_rho),abs(min_rho_data-min_rho),
-                     abs(max_rho_electrons_data-max_rho_electrons),
-                     abs(max_rho_protons_data-max_rho_protons))
-max_diff_number = max(abs(num_total_data-num_total),abs(num_electron_data-num_electron),
-                      abs(num_proton_data-num_proton),abs(num_photon_data-num_photon))
-max_diff_sum_weight = max(abs(sum_weight_total_data-sum_weight_total),
-                          abs(sum_weight_electron_data-sum_weight_electron),
-                          abs(sum_weight_proton_data-sum_weight_proton),
-                          abs(sum_weight_photon_data-sum_weight_photon))
-
-print('difference of field energy:', abs(EFyt-EF))
-print('tolerance of field energy:', 1.0e-3)
-print('difference of particle energy:', abs(EPyt-EP))
-print('tolerance of particle energy:', 1.0e-8)
-print('maximum difference of maximum electric field:', max_diffEmax)
-print('tolerance of maximum electric field difference:', 1.0e-9)
-print('maximum difference of maximum magnetic field:', max_diffBmax)
-print('tolerance of maximum magnetic field difference:', 1.0e-18)
-print('maximum difference of maximum charge density:', max_diffrhomax)
-print('tolerance of maximum charge density difference:', 1.0e-18)
-print('maximum difference of particle weight sum:', max_diff_sum_weight)
-print('tolerance of particle weight sum:', 0.5)
-
-assert(abs(EFyt-EF) < 1.0e-3)
-assert(abs(EPyt-EP) < 1.0e-8)
-assert(max_diffEmax < 1.0e-9)
-assert(max_diffBmax < 1.0e-18)
-assert(max_diffrhomax < 1.0e-18)
-assert(max_diff_number < 0.5)
-assert(max_diff_sum_weight < 0.5)
+
+# Field energy
+E2 = np.sum(Ex**2) + np.sum(Ey**2) + np.sum(Ez**2)
+B2 = np.sum(Bx**2) + np.sum(By**2) + np.sum(Bz**2)
+N  = np.array(ds.domain_width / ds.domain_dimensions)
+dV = N[0] * N[1] * N[2]
+values_yt['field energy'] = 0.5 * dV * (E2 * eps0 + B2 / mu0)
+
+# Max/min values of various grid quantities
+values_yt['maximum of Ex'] = np.amax(np.abs(Ex))
+values_yt['maximum of Ey'] = np.amax(np.abs(Ey))
+values_yt['maximum of Ez'] = np.amax(np.abs(Ez))
+values_yt['maximum of Bx'] = np.amax(np.abs(Bx))
+values_yt['maximum of By'] = np.amax(np.abs(By))
+values_yt['maximum of Bz'] = np.amax(np.abs(Bz))
+values_yt['maximum of |E|'] = np.amax(np.sqrt(Ex**2 + Ey**2 + Ez**2))
+values_yt['maximum of |B|'] = np.amax(np.sqrt(Bx**2 + By**2 + Bz**2))
+values_yt['maximum of rho'] = np.amax(rho)
+values_yt['minimum of rho'] = np.amin(rho)
+values_yt['electrons: maximum of rho'] = np.amax(np.abs(rho_electrons))
```

Comment:
```suggestion
values_yt['electrons: maximum of |rho|'] = np.amax(np.abs(rho_electrons))
```
I would write `|rho|` instead of `rho` here